### PR TITLE
feat(ui): Surface release in processing issues

### PR DESCRIPTION
**Pull request by matejminar**
This PR surfaces `release` and `distribution` in processing issues' UI.

image

Backend is being worked on here: https://github.com/getsentry/sentry/pull/46250

### CODE REVIEW

File: static/app/views/settings/project/projectProcessingIssues.tsx [hunk 1]
```diff
@@ -320,6 +323,14 @@ class ProjectProcessingIssues extends Component<Props, State> {
         {dsymUUID && <span> {dsymUUID}</span>}
         {dsymArch && <span> {dsymArch}</span>}
         {dsymName && <span> (for {dsymName})</span>}
+        {(release || dist) && (
+          <div>
+            <Tag tooltipText={t('Latest Release')}>
```
Comment (iambriccardo) [inline static/app/views/settings/project/projectProcessingIssues.tsx:328]: I don't know the content length for tooltips but this message sounds a bit too ambiguous. It can be that the release shown is not the latest.

I would rephrase it slightly:
```suggestion
            <Tag tooltipText={t('Latest Release Observed with Issue')}>
```

What do you think?

**Review by iambriccardo — APPROVED** [review]
LGTM, just a few minor comments regarding texts

File: static/app/views/settings/project/projectProcessingIssues.tsx (hunk 1 above)
Comment (matejminar) [reply static/app/views/settings/project/projectProcessingIssues.tsx:328]: A bit wordy, but I'll take it 👍
